operations/availabilityblocks.md

```diff
@@ -66,231 +62,222 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
+      "PickupDistribution": "AllInOneGroup",
+      "IsActive": false,
+      "QuoteId": null,
+      "AvailabilityBlockNumber": "478",
+      "ReleaseStrategy": "FixedRelease"
+    },
+    {
+      "Id": "c32386aa-1cd2-414a-a823-489325842fbe",
+      "EnterpriseId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+      "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
+      "RateId": "ed4b660b-19d0-434b-9360-a4de2ea42eda",
+      "VoucherId": null,
+      "BookerId": "ebd507c5-6bfd-4ca9-96aa-ffed6fa94f72",
+      "CompanyId": null,
+      "TravelAgencyId": null,
+      "Budget": {
+        "Currency": "USD",
+        "Value": 48,
+        "Net": null,
+        "Tax": null,
+        "TaxRate": null
+      },
+      "State": "Confirmed",
+      "ReservationPurpose": "Leisure",
+      "CreatedUtc": "2022-10-11T13:32:32Z",
+      "UpdatedUtc": "2022-10-11T13:32:32Z",
+      "FirstTimeUnitStartUtc": "2022-10-14T00:00:00Z",
+      "LastTimeUnitStartUtc": "2022-11-17T00:00:00Z",
+      "ReleasedUtc": null,
+      "RollingReleaseOffset": "P-3DT4H",
+      "ExternalIdentifier": "Block-0002",
+      "Name": "Rolling release",
+      "Notes": null,
+      "PickupDistribution": "AllInOneGroup",
+      "IsActive": false,
+      "QuoteId": "67eaf3c8-81e0-4ffb-b5f2-2b61803feb9c",
+      "AvailabilityBlockNumber": "479",
+      "ReleaseStrategy": "RollingRelease"
+    }
+  ],
+  "ServiceOrders": null,
+  "Adjustments": [
+    {
+      "Id": "e19297af-373e-4701-b4ea-afae0129bded",
+      "AvailabilityBlockId": "5ee074b1-6c86-48e8-915f-c7aa4702086f",
+      "ResourceCategoryId": "773d5e42-de1e-43a0-9ce6-f940faf2303f",
+      "StartUtc": null,
+      "EndUtc": null,
+      "FirstTimeUnitStartUtc": "2021-10-14T00:00:00Z",
+      "LastTimeUnitStartUtc": "2021-10-17T00:00:00Z",
+      "UnitCount": 6,
+      "ActivityState": "Active",
+      "ReleaseOverrideUtc": null,
+      "UpdatedUtc": "2021-10-21T13:32:32Z",
+      "IsActive": false,
+      "PaxCounts": null
+    }
+  ],
+  "Rates": null,
+  "Cursor": "c32386aa-1cd2-414a-a823-489325842fbe"
 }
 ```
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `AvailabilityBlocks` | array of [Availability block](#availability-block) | optional | Availability blocks. |
-| `ServiceOrders` | array of [Reservation](reservations.md#reservation-ver-2017-04-12) | optional | Service orders (for example reservations) linked to availability blocks. |
-| `Adjustments` | array of [Availability adjustment](availabilityadjustments.md#availability-adjustment) | optional | Availability adjustments of availability blocks. |
-| `Cursor` | string | optional | Unique identifier of the item one newer in time order than the items to be returned. If Cursor is not specified, i.e. null, then the latest or most recent items will be returned. |
+| `AvailabilityBlocks` | array of [Availability block](availabilityblocks.md#availability-block) | optional | Whether the response should contain the general availability blocks. |
```

Comment:
Why does it say "Whether the response should contain..."? This is not a boolean.